wp-content/plugins/core/src/Blocks/Types/Tabs/Tabs_Model.php

```diff
@@ -0,0 +1,67 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Blocks\Types\Tabs;
+
+use Tribe\Project\Blocks\Types\Base_Model;
+use Tribe\Project\Blocks\Types\Tabs\Tabs as Tabs_Block;
+use Tribe\Project\Templates\Models\Tab as Tab_Model;
+
+/**
+ * Class Model
+ *
+ * @package Tribe\Project\Blocks\Types\Tabs
+ */
+class Tabs_Model extends Base_Model {
+
+	/**
+	 * @return array
+	 */
+	public function get_data(): array {
+		return [
+			Tabs_Block::LAYOUT      => $this->get_layout(),
+			Tabs_Block::TITLE       => $this->get_title(),
+			Tabs_Block::DESCRIPTION => $this->get_description(),
+			Tabs_Block::TABS        => $this->get_tabs(),
+		];
+	}
+
+	/**
+	 * @return string
+	 */
+	private function get_layout(): string {
+		return $this->get( Tabs_Block::LAYOUT, Tabs_Block::LAYOUT_HORIZONTAL );
+	}
+
+	/**
+	 * @return string
+	 */
+	private function get_title(): string {
+		return $this->get( Tabs_Block::TITLE, '' );
+	}
+
+	/**
+	 * @return string
+	 */
+	private function get_description(): string {
+		return apply_filters( 'the_content', $this->get( Tabs_Block::DESCRIPTION, '' ) ); // TODO: Unsure why this get() doesn't apply the_content automatically.
```

Comment:
@msawicki notice my comment here. Curious to hear your thoughts on it.

Reply:
Wondering if it does now with the recent changes to return the `get_field()` value from ACF?

Reply:
Yep. Looks like it's working great now. Thanks!